xija/gui_fit/plots.py

```diff
@@ -161,27 +161,17 @@ def clearLayout(layout):
 
 
 def annotate_limits(limits, ax, dir='h'):
```

Comment:
Can you add a docstring? Also for the new functions later can you add `:param x: blah` and `returns: blah` for the public funcions?

Reply:
Done